src/foundry/foundry.js/pixi/containers/placeableObject.d.ts

```diff
@@ -1,425 +1,477 @@
+     * @param event - (unused)
+     */
+    protected _canUpdate(user: InstanceType<ConfiguredDocumentClass<typeof User>>, event?: any): boolean;
 
-  /**
-   * Does the User have permission to update the underlying Embedded Entity?
-   * @param event - (unused)
-   */
-  protected _canUpdate(user: User, event?: any): boolean;
+    /**
+     * Does the User have permission to delete the underlying Embedded Entity?
+     * @param event - (unused)
+     */
+    protected _canDelete(user: InstanceType<ConfiguredDocumentClass<typeof User>>, event?: any): boolean;
 
-  /**
-   * Does the User have permission to delete the underlying Embedded Entity?
-   * @param event - (unused)
-   */
-  protected _canDelete(user: User, event?: any): boolean;
+    /**
+     * Clear the display of the existing object
+     * @returns The cleared object
+     */
+    clear(): this;
 
-  /**
-   * Clear the display of the existing object
-   * @returns The cleared object
-   */
-  clear(): this;
+    /**
+     * Clone the placeable object, returning a new object with identical attributes
+     * The returned object is non-interactive, and has no assigned ID
+     * If you plan to use it permanently you should call the create method
+     *
+     * @returns A new object with identical data
+     */
+    clone(): this;
 
-  /**
-   * Clone the placeable object, returning a new object with identical attributes
-   * The returned object is non-interactive, and has no assigned ID
-   * If you plan to use it permanently you should call the create method
-   *
-   * @returns A new object with identical data
-   */
-  clone(): PlaceableObject<D>;
+    /** @override */
+    destroy(options?: Parameters<PIXI.Container['destroy']>[0]): void;
 
-  /**
-   * Draw the placeable object into its parent container
-   * @returns The drawn object
-   */
-  abstract draw(): Promise<this>;
+    /**
+     * Draw the placeable object into its parent container
+     * @returns The drawn object
+     */
+    abstract draw(): Promise<this>;
 
-  /**
-   * Draw the primary Sprite for the PlaceableObject
-   */
-  protected _drawPrimarySprite(texture: PIXI.Texture): PIXI.Sprite | null;
+    /**
+     * Refresh the current display state of the Placeable Object
+     * @returns The refreshed object
+     * @remarks `void` has been added because of `Drawing.refresh`
+     */
+    abstract refresh(): this | void;
 
-  /**
-   * Refresh the current display state of the Placeable Object
-   * @returns The refreshed object
-   */
-  abstract refresh(): unknown;
+    /**
+     * Register pending canvas operations which should occur after a new PlaceableObject of this type is created
+     * @param data    - (unused)
+     * @param options - (unused)
+     * @param userId  - (unused)
+     * @remarks Second variant has been added because of `Token._onCreate()`
+     */
+    protected _onCreate(
+      data: DeepPartial<DeepPartial<D>>,
```

Comment:
I think this should be `DeepPartial<D['data']['_source']>`. I know I hinted towards `ClientDocumentMixin#_onCreate` But apparently it's incorrect there, too, I guess I forgot that in the `_source` PR.
Also this will change slightly with #836.